packages/components/Callout/src/Callout.types.ts

```diff
@@ -199,8 +199,9 @@ export interface ICalloutProps extends IViewProps, ICalloutTokens {
   onShow?: () => void;
 
   /**
-   * @platform win32
-   * If true then the callout will attempt to focus the first focusable element that it contains.
+   * Determines whether the Callout sets focus when displayed.
+   * On macOS: this determines whether the Callout becomes the key window.
+   * On win32: If true then the callout will attempt to focus the first focusable element that it contains.
```

Comment:
For my own knowledge, does the prop behave differently between platforms?

Reply:
It does. It seems like on windows, it will put focus on the first focusable view inside the window. On macOS as far as I can tell, it's valid for the window to have first responder status (if it's a key window). So I didn't necessarily want to re-implement the windows behavior of having it focus on the first focusable window inside. I'd be open to making that change in the future.